src/controlCenter/widgets/mpris/mpris.vala

```diff
@@ -117,10 +136,11 @@ namespace SwayNotificationCenter.Widgets.Mpris {
         private void setup_mpris () throws Error {
             dbus_iface = Bus.get_proxy_sync (BusType.SESSION,
                                              "org.freedesktop.DBus",
-                                             "/org/freedesktop/DBus");
+                                             "/org/freedesktop/DBus");                              
```

Comment:
Remove whitespaces. I recommend using valas language servers built-in code formatter which fixes these issue for you :)